server/src/amyserver_tools/train_mlp.py

```
@@ -192,9 +206,27 @@ def load_json(path: Path) -> Optional[dict]:
 
 
 def flatten_landmarks_mean(frames: List[dict]) -> Optional[List[List[float]]]:
-    """Average landmarks across frames and return 42×3 list."""
+    """Average landmarks across frames with optional weighting and return 42×3 list.
```

Ruff (0.14.4)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)